EKF/ekf_helper.cpp

```diff
@@ -1649,7 +1648,7 @@ void Ekf::resetExtVisRotMat()
 	}
 
 	// reset the rotation matrix
-	_ev_rot_mat = quat_to_invrotmat(q_error);
+	_ev_rot_mat = Dcmf(q_error); // rotation from EV reference to EKF reference
```

Comment:
There is already a PR dealing with how the quat_to_invrotmat function is used - see https://github.com/PX4/ecl/pull/635